api/teachers/teacherRouter.js

```diff
@@ -56,31 +56,17 @@ router.post('/', async (req, res) => {
   }
 });
 
-router.put('/:id', (req, res) => {
+router.put('/:id', async (req, res) => {
   const teacher = req.body;
-  if (teacher) {
```

Comment:
ah good catch on repalcing my .then with async and await, i always fall back to it whenever I dont get the response I want